src/sentry/integrations/base.py

```diff
@@ -160,7 +160,11 @@ class Integration(object):
 
     def __init__(self, model, organization_id=None):
         self.model = model
-        self.organization_id = organization_id
+        if organization_id is not None:
+            self.org_integration = OrganizationIntegration.objects.get(
```

Comment:
should `self.org_integration` be set to None if organization_id is None? otherwise, won't you get an attribute error when you do the `if self.org_integration is None:` check below?